content/root/quick-start-guide.textile

```diff
@@ -205,6 +206,21 @@ blang[flutter].
   import 'package:ably_flutter/ably_flutter.dart' as ably;
   ```
 
+blang[go].
+  The Ably Client Library SDK is hosted on "Github":https://github.com/ably/ably-go and can be installed by running the @go get@ command:
+
+  ```[go]
+  go get -u github.com/ably/ably-go/ably
+  ```
+
+  Import the packages into your Go file:
+
+  bc[go]. import (
```

Comment:
The indentation here doesn't look consistent with the other Go samples on the page?

Reply:
Good spot... had issues with getting this to even format but think I've fixed it now!